.github/workflows/run_publish_package.yaml

```diff
@@ -0,0 +1,56 @@
+on: 
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Cron jobs are hard to read.  Consider adding a comment that gives when you expect this workflow to trigger.